src/components/Header.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Logo from '../assets/images/Logo.png'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch, faBars } from '@fortawesome/free-solid-svg-icons';
+
+import '../assets/styles/components/Header.css';
+
+// import { mockStudents } from './data';
+
+// const currentStudent = mockStudents[0];
+
+class Header extends React.Component{
+    state = {
+        isAuth: false,
+        searchInput: '',
+        isSearchCollapsed: true,
+        isMenuCollapsed: true
+    }
+
+    render(){
+        const handleChange = e => {
+            this.setState({searchInput: e.target.value})
+        }
+
+        const toggleSearchCollapse = () => {
+            if(!this.state.isMenuCollapsed) this.setState({isMenuCollapsed: !this.state.isMenuCollapsed});
+            this.setState({isSearchCollapsed: !this.state.isSearchCollapsed});
+        }
+
+        const toggleMenuCollapse = () => {
+            if(!this.state.isSearchCollapsed) this.setState({isSearchCollapsed: !this.state.isSearchCollapsed});
+            this.setState({isMenuCollapsed: !this.state.isMenuCollapsed});
+        }
+
+        return(
+            <header className="header">
+                <a href="/">
+                    <img className="header__logo" src={Logo} alt="Logo" />
+                </a>
+                <div className="header__search-container">
+                    <input onChange={handleChange} className="search-container__input" type="text" placeholder="Search" />
+                    <div className="search-container__icon-container">
+                        <FontAwesomeIcon icon={faSearch} />
+                    </div>
+                </div>
+
+                <div className="mobile-nav-buttons-container">
+                        <FontAwesomeIcon onClick={toggleSearchCollapse} icon={faSearch} />
```

Comment:
Fix indentation.

Reply:
Hello Germán, the indentation for Header.scss, Footer.scss and Header.js is now fixed